ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -958,30 +958,29 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-private-ovn
-- as: azure-ipi-private-p2-f14
-  cron: 29 12 8,23 * *
+- as: azure-ipi-private-workers-marketimage-p2-f14
+  cron: 33 12 15,30 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      E2E_RUN_TAGS: '@azure-ipi and @disconnected and @network-openshiftsdn and not
-        @fips'
+      E2E_RUN_TAGS: '@azure-ipi and @network-openshiftsdn and not @fips'
       TAG_VERSION: '@4.10'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-private-p3-f28-destructive
-  cron: 42 20 14 * *
+    workflow: cucushift-installer-rehearse-azure-ipi-private-workers-marketimage
+- as: azure-ipi-private-workers-marketimage-p3-f28-destructive
+  cron: 57 23 7 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@azure-ipi and @disconnected and @network-openshiftsdn and not
```

Comment:
I guess we can drop `@disconnected` together.

Reply:
thanks for reveiw, missed in some job config, should be dropped now on all related jobs config.